### DISCUSSION TITLE
Adding folders in order to organize the habits

### DISCUSSION THREAD

**Discussion by chromer030** [iSoron/uhabits#1659] [Feature Requests]
By adding support for folders , long list of habits can be organized specially archived ones.

**Comment by BADF00D** [comment]
I like to have this feature, too, Currently I use the same color for habits that belong to the same group/folder. But it would be nicer to have direct support for it.

**Comment by lkraav** [comment]
Duplicate https://github.com/iSoron/uhabits/discussions/495
